package/MDAnalysis/core/selection.py

```diff
@@ -173,6 +175,7 @@ def __init__(cls, name, bases, classdict):
         type.__init__(type, name, bases, classdict)
         try:
             _SELECTIONDICT[classdict['token']] = cls
+            _SELECTIONDICT[classdict['token'].lower()] = cls
```

Comment:
This is because the singular for altlocs is `altLoc` but the previous selection keyword was "altloc".